code/game/objects/effects/spawners/random_spawners.dm

```diff
@@ -304,11 +303,6 @@
 		/turf/simulated/wall/mineral/plastitanium/nodiagonal = 2,
 		/obj/structure/falsewall/plastitanium = 2)
 
-/obj/effect/spawner/random_spawners/syndicate/layout/door/secret
-	name = "50pc falsewall 50pc wall"
-	result = list(/turf/simulated/wall/mineral/plastitanium/nodiagonal = 1,
-		/obj/structure/falsewall/plastitanium = 1)
-
```

Comment:
Why did you remove this falsewall spawner?

Reply:
Because it isn't used anymore. The only place it was used was in the red-boxed area on the right, around the (now also removed) plasma tank. Those tiles are solid walls now. The object was only defined so it could be used there. Not it isn't used there, it has no justification to exist.